newsboat.cpp

```diff
@@ -26,7 +26,8 @@ extern "C" {
 using namespace newsboat;
 
 void print_usage(const std::string& argv0, const std::string& config_path,
-	const std::string& urls_path, const std::string& cache_path)
+	const std::string& urls_path, const std::string& cache_path, const std::string& queue_path,
+	const std::string& search_path, const std::string& cmdline_path)
```

Comment:
Apparently this function needs all the paths, so let's replace all these arguments with `const ConfigPaths&`.